resources/definitions/ultimaker_s8.def.json

```diff
@@ -447,7 +447,8 @@
         "retraction_hop_enabled": { "value": true },
         "retraction_min_travel": { "value": "2.5 if support_enable and support_structure=='tree' else line_width * 2.5" },
         "retraction_prime_speed": { "value": 15 },
-        "roofing_monotonic": { "value": false },
+        "roofing_extension": { "value": 2 },
```

Comment:
Having a fixed value can lead to uncovered borders, e.g. if you have a 0.8mm nozzle and 4 walls. But that may be an extreme case that we can ignore.